cmake/OpenCVFindLibsVideo.cmake

```diff
@@ -217,7 +217,11 @@ ocv_clear_vars(HAVE_FFMPEG)
 if(WITH_FFMPEG)
   if(WIN32 AND NOT ARM)
     include("${OpenCV_SOURCE_DIR}/3rdparty/ffmpeg/ffmpeg.cmake")
-    set(HAVE_FFMPEG TRUE)
+    download_win_ffmpeg(res script)
```

Comment:
same note about combined return vars. `script` is enough (probably renamed to `__ffmpeg_script`).